services/outputhost/cgcache.go

```diff
@@ -264,7 +264,7 @@ func newConsumerGroupCache(destPath string, cgDesc shared.ConsumerGroupDescripti
 		cfgMgr:                   h.cfgMgr,
 	}
 
-	cgCache.checkSingleCGVisibleCache = cache.NewLRUWithInitialCapacity(1, 1 << 6)
+	cgCache.checkSingleCGVisibleCache = cache.NewLRUWithInitialCapacity(1, 1<<6)
```

Comment:
nit: 64 is more readable than 1<<6.. I know you didn't make this change, but isn't 64 too big for this cache ?